src/backend/marsha/core/management/commands/send_reminders.py

```diff
@@ -68,6 +68,7 @@ def send_reminders_and_update_liveregistrations_step(
                 vars = {
                     "email": liveregistration.email,
                     "time_zone": settings.TIME_ZONE,
+                    "unregister_link": liveregistration.get_cancel_send_reminder_url(),
```

Comment:
```suggestion
                    "cancel_send_reminder_url": liveregistration.get_cancel_send_reminder_url(),
```
IMHO, it would be clearer if those names reflects each other.